fe/fe-core/src/main/java/org/apache/doris/catalog/Env.java

```diff
@@ -688,7 +690,11 @@ public Env(boolean isCheckpointCatalog) {
         this.brokerMgr = new BrokerMgr();
         this.resourceMgr = new ResourceMgr();
 
-        this.globalTransactionMgr = new GlobalTransactionMgr(this);
+        if (Config.isCloudMode()) {
```

Comment:
EnvFactory.getInstance().createGlobalTransactionMgr(this);

EnvFactory and CloudEnvFactory need implement createGlobalTransactionMgr